javascript/javascript1/week4/creditCardNo/cc.js

```diff
@@ -0,0 +1,13 @@
+function formatCreditCardNumber(number) {
+    number = number.toString();
+    const gap_size = 4;
+    let result = "";
+    while (number.length > 0) // Loop through string
```

Comment:
technically this will work. Another way would be by creating an array, and using array.join(' ')